evm/chains/sharding/mainchain_handler/config.py

```diff
@@ -0,0 +1,5 @@
+DEPOSIT_SIZE = 10 ** 20
```

Comment:
It would be good to have a comment explaining this *magic* number.

Reply:
Have done it:) Thank you